mmaction/datasets/activitynet_dataset.py

```diff
@@ -166,6 +175,48 @@ def dump_results(self, results, out, output_format, version='VERSION 1.3'):
                 'external_data': {}
             }
             mmcv.dump(output_dict, out)
+
+            if classifier_input:
+                with open(out) as f_out:
```

Comment:
Can this io process be skipped? As you dump `output_dict` into `out`, and now read it again into a new variable `proposal_results`

Reply:
Maybe not?
`out` is used to evaluate the performance of the proposal generator and `classifier_input` is fed into the classifier.